update readme (#86)
aternosorg/modbot: README.md

### Setup

1. Install [Node.js](https://nodejs.org/en/) and [Mysql](https://www.mysql.com/)
2. create a user and databse for the bot

> 3. Clone or download the repository

Reviewer: Add `npm install` as a step

4. create a discord Application [here](https://discordapp.com/developers/applications/)
5. add a Bot to it and copy the auth token
6. copy the example.config.json to config.json and enter the auth token and mysql login
7. run this command in the directory

   ```bash
   node bot.js
   ```

> 8. to invite the bot to your server replace ID with your applications Client ID https://discordapp.com/oauth2/authorize?client_id=ID&scope=bot&permissions=11264 and open the link

Reviewer: ```suggestion
8. To invite the bot to your server replace `ID` with the client ID of your application https://discordapp.com/oauth2/authorize?client_id=ID&scope=bot&permissions=11264 and open the link
```

> 9. (optional) Set up a logchannel by using ,logchannel #channel

Reviewer: ```suggestion
9. (optional) Set up a log channel by using !logchannel #channel
```

### Setup

1. Install [Node.js](https://nodejs.org/en/download/) and [MySQL](https://dev.mysql.com/downloads/mysql/)
2. Create a MySQL user and database for the bot
3. Clone or download the repository

> 4. run `npm install`

Reviewer: ```suggestion
4. Run `npm install`
```
